CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 Changelog
 =========
+1.24.0 (2022-02-27)
+-------------------
+MLflow 1.24.0 includes several major features and improvements:
+
+Features:
+
+- [Tracking] Support uploading, downloading, and listing artifacts through the MLflow server via ``mlflow server --serve-artifacts`` (#5320, @BenWilson2, @harupy)
+- [Tracking] Add the ``registered_model_name`` argument to ``mlflow.autolog()`` for automatic model registration during autologging (#5395, @WeichenXu123)
+- [UI] Improve and restructure the Compare Runs page. Additions include "show diff only" toggles and scrollable tables (#5306, @WeichenXu123)
+- [Models] Introduce ``mlflow.pmdarima`` flavor for pmdarima models (#5373, @BenWilson2)
+- [Models] When loading an MLflow Model, print a warning if a mismatch is detected between the current environment and the Model's dependencies (#5368, @WeichenXu123)
+- [Models] Support computing custom scalar metrics during model evaluation with ``mlflow.evaluate()`` (#5389, @MarkYHZhang)
```

Comment:
@jinzhang21 @MarkYHZhang Here's the release notes entry for custom scalar metrics. PTAL before the release occurs on Sunday, 2/27. Apologies for the late-Friday ping.

Reply:
Looks good to me. Thanks! 🙏